manifests/server.pp

```diff
@@ -55,6 +55,9 @@
   $manage_firewall          = $puppetdb::params::manage_firewall,
   $java_args                = $puppetdb::params::java_args,
   $max_threads              = $puppetdb::params::max_threads,
+  $command_threads          = $puppetdb::params::command_threads,
```

Comment:
Since these all default to undef, you could probably just explicitly set them to undef here.